wordpress-mobile/WordPress-Android
# Update app icons to new blue color

Fixes #14028. This PR updates the app's icons to use the new blue color we're using in the rest of the UI.

**To test**

- Build and run, and verify the icons have the new light appearance on high resolution and low resolution, and look correct.

## Regression Notes
1. Potential unintended areas of impact

None, just updating resources.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
